modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java

```diff
@@ -531,12 +531,16 @@ protected void preparePatch(
 	private ServiceContext _createServiceContext(Organization organization)
 		throws Exception {
 
-		return ServiceContextRequestUtil.createServiceContext(
+		ServiceContext serviceContext = ServiceContextFactory.getInstance(
```

Comment:
Since there are a lot of uses of `ServiceContextRequestUtil`... do you think it would make sense to add this change inside `ServiceContextRequestUtil.createServiceContext` so that everybody gets the right data?

Reply:
👍